pkgs/tools/misc/adrgen/default.nix

```diff
@@ -11,12 +12,21 @@ buildGoModule rec {
 
   src = fetchFromGitHub {
     owner = "asiermarques";
-    repo = "adrgen";
+    repo = pname;
```

Comment:
I'd prefer for this to stay as it was before, its nicer to read IMO.

Reply:
Sure, I reverted this change.